### DISCUSSION TITLE
刚创建一个taro小程序，报了一个找不到sass类型定义文件

### DISCUSSION THREAD

**Discussion by lyj6464** [NervJS/taro#14107] [Q & A]
刚创建一个taro小程序，tsconfig.json就报了ts错误
![image](https://github.com/NervJS/taro/assets/121071588/de5542c1-7009-4c16-be8b-7983a633c513)
这个怎么解决，虽然不影响项目运行，但感觉怪怪的

**Comment by 18338625417** [comment]
借楼同问 taro3.6.10 这个版本 dev:weapp 起不来项目

**Comment by souldar** [comment]
应该是因为 @types/sass 这个包安装成了 1.45.0 这个版本，它删除了所有的类型声明文件，而 taro 的一些版本依赖的 sass 版本又比较低，还没有自己的类型文件，所以导致找不到 sass 类型文件。

我看最新的代码里已经锁死了 @types/sass 这个版本到1.43.1，所以升级taro应该可以解决问题。但不排除需要删掉项目 lock 文件重装依赖，或者使用 resolutions 强制指定依赖。

最终问题好没好要看项目实际安装的 @types/sass 是不是 1.43.1

**Reply by 18338625417** [reply]
我项目中没有安装 @types/sass这个包也会报这个 不知道是不是@nutui/nutui-react-taro 这个第三方组件库中包含了

**Reply by souldar** [reply]
taro 会引的，可以通过 yarn why @types/sass 看到依赖信息
